### PR TITLE
Added audio upload support

### PR DESCRIPTION
**Pull request by sahil97**
#### Definition of Done
- [ ] Acceptance criteria has been met
- [ ] Postman endpoint with example has been created

### CODE REVIEW

File: frontend/app_v2/src/common/constants/docTypes.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const DOC_IMAGE = 'FVPicture'
 export const DOC_VIDEO = 'FVVideo'
 export const SUPPORTED_IMAGE_EXTENSIONS = ['jpeg', 'png', 'gif', 'tiff']
```
Comment (sarahfirstvoices) [inline frontend/app_v2/src/common/constants/docTypes.js:14]: I know this isn't part of this review, but does uppy handle variations on these file extensions, like jpg and tif?

Comment (sahil97) [reply frontend/app_v2/src/common/constants/docTypes.js:14]: Good point. I can check

**Review by sarahfirstvoices — CHANGES_REQUESTED** [review]
Best solution is probably to switch from file extensions to mime types

File: frontend/app_v2/src/common/constants/docTypes.js (hunk 1 above)
Comment (sahil97) [reply frontend/app_v2/src/common/constants/docTypes.js]: Added extensions manually instead of the mime type to be more specific.

**Review by sarahfirstvoices — APPROVED** [review]

**Pull request merged**